### PR TITLE
Fix msg size too large

### PR DESCRIPTION
**Pull request by fraidev**
Related: https://github.com/infinyon/poc-siemonster/issues/1

### CODE REVIEW

**Comment by fraidev** [comment]
Do I need to bump the version?

File: crates/kafka-source/src/source.rs [hunk 1]
```diff
@@ -3,14 +3,15 @@ use async_std::channel::{self, Sender};
...
-use fluvio_connector_common::tracing::{info, trace};
+use fluvio_connector_common::tracing::{error, info, trace};
 use fluvio_connector_common::Source;
 use futures::{stream::LocalBoxStream, StreamExt};
 use kafka::consumer::{Consumer, FetchOffset, GroupOffsetStorage};
 
 use crate::config::KafkaConfig;
 
 const CHANNEL_BUFFER_SIZE: usize = 10000;
+const DEFAULT_FETCH_MAX_BYTES_PER_PARTITION: i32 = 1024 * 1024;
```
Comment (fraidev) [inline crates/kafka-source/src/source.rs:14]: Is this good for a default? The Kafka library uses 32 × 1024.

Comment (sehz) [reply crates/kafka-source/src/source.rs:14]: should be good

**Review by sehz — CHANGES_REQUESTED** [review]

File: crates/kafka-source/src/source.rs [hunk 2]
```diff
@@ -47,15 +53,21 @@ impl KafkaSource {
 impl<'a> Source<'a, Record> for KafkaSource {
     async fn connect(self, _offset: Option<Offset>) -> Result<LocalBoxStream<'a, Record>> {
         let (sender, receiver) = channel::bounded(CHANNEL_BUFFER_SIZE);
```
Comment (sehz) [inline crates/kafka-source/src/source.rs:55]: don't need channel here.  but it's not show stopper

**Comment by digikata** [comment]
re: bumping the version - I just merged your other pr which bumps the version - after this pr is ready to merge, we can then publish the kafka-source connector to the hub at 0.2.6

File: crates/kafka-source/src/backoff.rs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+use std::time::Duration;
+
```
Comment (sehz) [inline crates/kafka-source/src/backoff.rs:2]: this is used in fluvio https://crates.io/crates/adaptive_backoff

**Review by sehz — APPROVED** [review]
LGTM

**Pull request merged**